### DISCUSSION TITLE
Feature merge by automated tool

### DISCUSSION THREAD

**Discussion by james-yoo** [rapyuta-robotics/RapyutaSimulationPlugins#218] [General]
Hi Rapyuta-Robotics team!
Are there any plans to implement a devops pipline (e.g use github webhook) to automatically merge features committed to the development branch into each branch of UE? It is inefficient to manually merge features into each branch.

**Comment by yuokamoto** [comment]
Working on github action to create PR automatically
https://github.com/rapyuta-robotics/rclUE/pull/129

**Reply by james-yoo** [reply]
@yuokamoto 
Super cool!

**Comment by james-yoo** [comment]
@yuokamoto 
Thanks for your support. Are you planning to apply the github action to this repo(RapyutaSimulationPlugins) as well?

**Comment by yuokamoto** [comment]
yes done

**Comment by james-yoo** [comment]
@yuokamoto,
As far as I know, the 'devel' branch is the maintenance branch of this repo, so shouldn't it cherry-pick UE5.2 and UE5.3 if the PR is merged into the devel branch as below? 

name: Cherry pick and create PR
  if: **github.base_ref == 'devel'** && matrix.version != github.base_ref

**Reply by yuokamoto** [reply]
I think
- if you dev/test feature in UE5.2 and/or UE5.3, most of the case it should be merged to devel as well. I guess small number of changes are specific to UE 5.2 and/or UE5.3.
- Since this action is just create PR, you can close without merge if that feature is specific to UE5.2 and UE5.3

**Reply by james-yoo** [reply]
Understood. Thanks.
